test/Renci.SshNet.IntegrationTests/AuthenticationTests.cs

```diff
@@ -90,18 +90,15 @@ public void Multifactor_PublicKeyWithPassPhrase()
         }
 
         [TestMethod]
-        [ExpectedException(typeof(SshPassPhraseNullOrEmptyException))]
         public void Multifactor_PublicKeyWithEmptyPassPhrase()
```

Comment:
Could you delete this test or move it to PasswordAuthenticationMethodTest.cs in the unit tests project? Not sure why it's an integration test

Reply:
`Test_PrivateKey_SSH2_Encrypted_ShouldThrowSshPassPhraseNullOrEmptyExceptionWhenPassphraseIsNull` in `PrivateKeyFileTest.cs` already covers this, so I removed it.